src/calendar/calendar.tsx

```diff
@@ -398,10 +424,32 @@ export const Component = (
     const renderCancelDoneButton = () => {
         if (type === "standalone" || !withButton) return;
 
+        let isDisabled = true;
+        const isDayView = ["default"].includes(currentView);
+
+        switch (variant) {
+            case "single":
+                isDisabled = selectedStartDate?.length ? false : true;
+                break;
+            case "range":
+                isDisabled = ![selectedStartDate, selectedEndDate].every(
```

Comment:
Oops, it may not work correctly.

button disabled when both value didn't been selected yet.
`isDisabled` value  `true`

button active when both value been selected
`isDisabled`  value `false`

so when 
!selectedStartDate &&  !selectedEndDate  
          true                                  false              // return false, button active
          true                                  true               // correct one -> both value selected, button active

Reply:
ah, it should have been `!selectedStartDate || !selectedEndDate`